app/serializers/app.js

```diff
@@ -1,6 +1,15 @@
 import ApplicationSerializer from './application';
 
 export default ApplicationSerializer.extend({
+  normalize() {
```

Comment:
Ya, this looks good (and likely works around the problem better across the board).